From 6331d12bc4814a27579b96acc299314d278be4dc Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Fri, 25 Sep 2020 10:00:23 +0100
Subject: [PATCH 03/13] Set RAM to something larger than qemu default.

The qemu default is quite small - 128MB - so may be unsuitable on many
architectures even though we are only booting a kernel and trivial
initrd.  It's better to set this to something more normal, and in case
we don't use the extra RAM it won't make any difference.
---
 qemu-sanity-check.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/qemu-sanity-check.in b/qemu-sanity-check.in
index 5a3594a..5066ebc 100644
--- a/qemu-sanity-check.in
+++ b/qemu-sanity-check.in
@@ -28,6 +28,7 @@ verbose=no
 timeout=10m
 accel=kvm:tcg
 cpu=
+memory=768
 machine=
 
 # Both libguestfs and virt-manager choose cpu=host when we think that
@@ -189,6 +190,8 @@ if [ "$cpu" != "" ]; then
     argv[$((i++))]="-cpu"
     argv[$((i++))]="$cpu"
 fi
+argv[$((i++))]="-m"
+argv[$((i++))]="$memory"
 argv[$((i++))]="-no-reboot"
 argv[$((i++))]="-serial"
 argv[$((i++))]="file:$test_output"
-- 
2.43.0

